src/translations/index.js

```diff
@@ -239,12 +269,40 @@ const TRANSLATIONS = {
         'validate-words-text': 'Bitte wähle das richtige Wort aus deiner Liste von Wiederherstellungswörtern aus.',
         'validate-words-back': 'Zurück zu den Wörtern',
         'validate-words-skip': 'Überprüfung erstmal überspringen',
+        'validate-words-1-hint': 'Was ist das 1te Wort?',
+        'validate-words-2-hint': 'Was ist das 2te Wort?',
+        'validate-words-3-hint': 'Was ist das 3te Wort?',
+        'validate-words-4-hint': 'Was ist das 4te Wort?',
+        'validate-words-5-hint': 'Was ist das 5te Wort?',
+        'validate-words-6-hint': 'Was ist das 6te Wort?',
+        'validate-words-7-hint': 'Was ist das 7te Wort?',
+        'validate-words-8-hint': 'Was ist das 8te Wort?',
+        'validate-words-9-hint': 'Was ist das 9te Wort?',
+        'validate-words-10-hint': 'Was ist das 10te Wort?',
+        'validate-words-11-hint': 'Was ist das 11te Wort?',
+        'validate-words-12-hint': 'Was ist das 12te Wort?',
+        'validate-words-13-hint': 'Was ist das 13te Wort?',
+        'validate-words-14-hint': 'Was ist das 14te Wort?',
+        'validate-words-15-hint': 'Was ist das 15te Wort?',
+        'validate-words-16-hint': 'Was ist das 16te Wort?',
+        'validate-words-17-hint': 'Was ist das 17te Wort?',
+        'validate-words-18-hint': 'Was ist das 18te Wort?',
+        'validate-words-19-hint': 'Was ist das 19te Wort?',
+        'validate-words-20-hint': 'Was ist das 20te Wort?',
+        'validate-words-21-hint': 'Was ist das 21te Wort?',
+        'validate-words-22-hint': 'Was ist das 22te Wort?',
+        'validate-words-23-hint': 'Was ist das 23te Wort?',
+        'validate-words-24-hint': 'Was ist das 34te Wort?',
 
         'export-file-heading': 'Schlüsseldatei herunterladen',
+        'export-file-intro-heading': 'Sichere deinen Account',
+        'export-words-intro-heading': 'Es gibt keine Passwort Wiederherstellung',
         'export-button-words': 'Wiederherstellungswörter anzeigen',
         'export-button-file': 'Schlüsseldatei herunterladen',
         'export-finish': 'Export abschließen',
         'export-more-options-heading': 'Zusätzliche Export Optionen',
```

Comment:
Exportoptionen :)

Reply:
no longer used, so I removed it.

Reply:
I think we need to check on the translation file in terms of which strings are even still in use.